### PR TITLE
getting song time from canvas to redux and PlayerBar

### PR DESCRIPTION
**Pull request by YogiHa**
getting song time from canvas to redux store, and after passing it to PlayerBar component. uncomment the console.log in PlayerBar to see the result

also try to fix Travis issue with grpc dependency

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "dotenv": "^8.2.0",
     "emailjs-com": "^2.4.0",
     "firebase": "^7.1.0",
+    "grpc": "^1.24.2",
```
Comment (ArnasDickus) [inline package.json:9]: It seems your Travis test fix solution is not working

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, thanks for adding PR.
There are some console warnings. Also could you fix sectionThreeIcons.js erros. Contributor who supposed to fixed got stuck.
console errros

Also Playing a song get's weird animation. I tested with current development branch. It looks normal.  So it must be something on your end.
playing-song-wierd-animation

**Comment by ArnasDickus** [comment]
@YogiHa Travis bug was fixed in [PR #73](https://github.com/zero-to-mastery/visual-music/pull/73). Please update your branch.

**Comment by YogiHa** [comment]
@ArnasDickus I will update my package.json as follow PR 73,

and will push when I'll manage to fix sketch, every second the sketch exporting for some reason it restrating

**Comment by YogiHa** [comment]
opening a new PR due to reworking on this task from other angle

**Pull request closed without merging**